modules/test/playwright/pages/object-web/object-entries/ViewObjectEntriesPage.ts

```diff
@@ -284,4 +290,62 @@ export class ViewObjectEntriesPage {
 		await this.goto(objectDefinition);
 		await this.objectEntryButton.click();
 	}
+
+	async schedulePropertiesCrud(scheduleProperties: ScheduleProperty[]) {
+		for (const scheduleProperty of scheduleProperties) {
+			await this.page
+				.getByLabel(
+					`Never ${scheduleProperty === 'Expiration' ? 'Expire' : 'Review'}`,
+					{exact: true}
+				)
+				.uncheck();
+
+			const date = new Date();
+
+			// Since expiration does not accept current time, we add 200 minutes to the current time
+
+			date.setMinutes(date.getMinutes() + 200);
+
+			await this[`${scheduleProperty.toLowerCase()}DateInput`].fill(
+				getObjectEntryUIDateTimeFormat(date)
+			);
+
+			await this.choosePublicationOption('publish');
+
+			await waitForAlert(this.page);
+
+			await expect(
+				this[`${scheduleProperty.toLowerCase()}DateInput`]
+			).toHaveValue(getObjectEntryUIDateTimeFormat(date));
+
+			date.setDate(date.getDate() + 1);
+
+			const tomorrow = getObjectEntryUIDateTimeFormat(date);
+
+			await this[`${scheduleProperty.toLowerCase()}DateInput`].fill(
+				tomorrow
+			);
+
+			await this.choosePublicationOption('publish');
+
+			await waitForAlert(this.page);
+
+			await expect(
+				this[`${scheduleProperty.toLowerCase()}DateInput`]
+			).toHaveValue(tomorrow);
+
+			await this.page
+				.getByLabel(
+					`Never ${scheduleProperty === 'Expiration' ? 'Expire' : 'Review'}`,
+					{exact: true}
+				)
+				.check();
+
+			await this.choosePublicationOption('publish');
+
+			await expect(
+				this[`${scheduleProperty.toLowerCase()}DateInput`]
+			).toHaveValue('');
+		}
+	}
```

Comment:
Hey @rtienpal, I believe we can find a middle ground between creating functions to avoid code repetition and maintaining the clarity of our test.

Imagine if all tests had most of their steps inside functions in other files, it would be very difficult to perform a progressive reading of a set of tests to understand their flow.

When you are available, we can talk more about this.

Reply:
Also, in most cases, we have kept the expects within the body of the tests following this same idea of ​​clarity.